science/openmpi/Portfile

```diff
@@ -136,15 +137,13 @@ if {${subport} != ${name}} {
     archive_sites
 
     depends_build-append        port:pkgconfig
+    depends_build-append        port:perl5
     depends_lib-append          port:hwloc
     depends_run                 port:mpi_select port:mpi-doc
     select.group                mpi
     select.file                 ${filespath}/${name}-${cname}
     livecheck.type              none
 
-    # avoid "Can't open profile/psizeof_f08.f90 for writing"
-    use_parallel_build no
```

Comment:
I agree if this is no longer needed it would be good to reenable parallel builds.

Reply:
It would be useful though if you could find reference to an upstream commit addressing this ... Just to prove its specifically been addressed.

Reply:
I don't have much context around this change, actually: I just never saw it occur in the dozens of builds I ran and not making this change would have made the take many times longer…

Reply:
OK thanks. I am inclined to agree. If problems reappear the option can always be put back.